modules/auxiliary/scanner/http/rdp_web_login.py

```diff
@@ -108,30 +108,30 @@ def check_login(rhost, rport, targeturi, domain, username, password, timeout, us
     The timeout is used to specify the amount of milliseconds where a
     response should consider the username invalid."""
 
-    url = f'https://{rhost}:{rport}/{targeturi}'
-    body = f'DomainUserName={domain}%5C{username}&UserPass={password}'
+    url = 'https://' + rhost + ':' + rport + '/' + targeturi
+    body = 'DomainUserName=' + 'domain' + '%5C' + username + '&UserPass=' + password
     headers = {'Host':rhost,
                'User-Agent': user_agent,
                'Content-Type': 'application/x-www-form-urlencoded',
-               'Content-Length': f'{len(body)}',
-               'Origin': f'https://{rhost}'}
+               'Content-Length': str(len(body)),
+               'Origin': 'https://' + rhost}
     session = requests.Session()
     report_data = {'domain':domain, 'address': rhost, 'port': rport,
                    'protocol': 'tcp', 'service_name':'RDWeb'}
     try:
         request = session.post(url, data=body, headers=headers,
                                timeout=(timeout / 1000), verify=False, allow_redirects=False)
         if request.status_code == 302:
-            module.log(f'Login {domain}\\{username}:{password} is valid!', level='good')
+            module.log('Login ' + domain + '\\' + username + ':' + password + ' is valid!', level='good')
             module.report_correct_password(username, password, **report_data)
         elif request.status_code == 200:
-            module.log(f'Password {password} is invalid but {domain}\\{username} is valid! Response received in {request.elapsed.microseconds / 1000} milliseconds',
+            module.log('Password ' + password + ' is invalid but ' + domain + '\\' + username + ' is valid! Response received in ' + str(request.elapsed.microseconds / 1000) + ' milliseconds',
                        level='good')
             module.report_valid_username(username, **report_data)
         else:
-            module.log(f'Received unknown response with status code: {request.status_code}')
+            module.log('Received unknown response with status code: ' + request.status_code)
     except requests.exceptions.Timeout:
-        module.log(f'Login {domain}\\{username}:{password} is invalid! No response received in {timeout} milliseconds',
+        module.log('Login ' + domain + '\\' + username + ':' + password + ' is invalid! No response received in ' + timeout + ' milliseconds',
```

Comment:
this timeout needs a str() wrapper to get the concatenation to work. So replace with str(timeout)